src/naive/datetime/mod.rs

```diff
@@ -987,7 +987,7 @@ impl NaiveDateTime {
     /// ```
     #[inline]
     pub const fn with_minute(&self, min: u32) -> Option<NaiveDateTime> {
-        Some(NaiveDateTime { time: try_opt!(self.time.with_minute(min)), ..*self })
+        Some(NaiveDateTime { time: try_opt!(ok!(self.time.with_minute(min))), ..*self })
```

Comment:
Why don't we have a `try_ok_opt!()` or whatever macro?

Reply:
They could have use that macro also. It was only there to split up the commits, and replaced in a later commit.